migrations/versions/407e74de5553_adding_user_table.py

```diff
@@ -8,14 +8,13 @@
 
 # revision identifiers, used by Alembic.
 revision = '407e74de5553'
-down_revision = '3a5eba38e4e8'
+down_revision = '12f9dcc2e36c'
```

Comment:
Agreed, the whole first commit looks dodgy, it adds the `Users` model again. Could this be down to a rebase?